react/components/SearchBar/index.js

```diff
@@ -27,6 +27,7 @@ function SearchBarContainer(props) {
     }),
     autocompleteAlignment = 'right',
     openAutocompleteOnFocus = false,
+    containerMode = 'overlay' | 'container',
```

Comment:
I suggest to remove this operator since that's a JS file and not TS, it might have unexpected results.

```suggestion
    containerMode = 'overlay',
```

Reply:
Thanks man this suggestion has already updated